test/index.test.js

```diff
@@ -261,3 +262,81 @@ QUnit.test('duration returns NaN, Infinity or duration according to the HTML sta
               'duration returns duration property when readyState' +
               ' and duration property are both higher than 0');
 });
+
+QUnit.test('getVideoPlaybackQuality API exists', function(assert) {
+  const propertyCalls = [];
+  const videoPlaybackQuality = { test: 'test' };
+  const mockFlash = {
+    el_: {
+      /* eslint-disable camelcase */
+      vjs_getProperty(attr) {
+        propertyCalls.push(attr);
+        return videoPlaybackQuality;
+      }
+      /* eslint-enable camelcase */
+    }
+  };
+
+  assert.equal(typeof Flash.prototype.getVideoPlaybackQuality,
```

Comment:
Can do. Should it also be removed from https://github.com/videojs/video.js/pull/4286/files#diff-e4abe4634ade6e256f338fe0c23f6517R280 ?